web/src/pages/Route/Create.tsx

```diff
@@ -93,6 +95,10 @@ const Page: React.FC<Props> = (props) => {
     }
   }, []);
 
+  const getProxyrewriteEnable =() => {
```

Comment:
This should be `getProxyRewriteEnable` ?

Reply:
More like a spelling  issue 🤔 but not hurry

Reply:
fixed, thanks